ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingConfig.java

```diff
@@ -55,6 +55,12 @@ public class OptimizingConfig {
   //base.file-index.hash-bucket
   private int baseHashBucket;
 
+  //self-optimizing.trigger.base-max-delay
```

Comment:
Would it be more appropriate to name the property `self-optimizing.trigger.max-base-delay`?

Reply:
Yes, it is better. I will change them to `max-base-delay` and `max-hive-delay`.